transforms/images/roi-relabel-plugin/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "polus-plugins-transforms-images-roi-relabel"
+version = "0.2.0"
+description = ""
+authors = ["Najib Ishaq <najib.ishaq@nih.gov>"]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.9.7 || >3.9.7,<3.12"
+bfio = { version = "2.1.9", extras = ["all"] }
+filepattern = "^2.2.7"
+typer = { version = "^0.7.0", extras = ["all"] }
+numpy = "^1.24.2"
+scipy = "^1.10.0"
+networkx = "^3.0"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.2.1"
+pytest-cov = "^4.0.0"
+pytest-sugar = "^0.9.6"
+pytest-xdist = "^3.2.0"
+pytest-benchmark = "^4.0.0"
+bump2version = "^1.0.1"
+pre-commit = "^3.0.4"
+flake8 = "^6.0.0"
+flake8-docstrings = "^1.7.0"
+black = "^23.1.0"
+mypy = "^1.0.0"
+
+[tool.poetry.group.examples.dependencies]
```

Comment:
Can you add these packages to tool.poetry.group.dev.dependencies as we just want to have poetry.dependencies and poetry.group.dev.dependencies only

Reply:
Done